app/renderer/views/Home/index.js

```diff
@@ -1,23 +1,27 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import { Link } from 'react-router-dom'
+import AppHeader from '../../components/AppHeader'
 import Icon from '../../components/Icon'
 
 import { StyledHome } from './styles.js'
 
 function Home (props) {
   return (
-    <StyledHome>
-      <div className='banner'>
-        <h1 className='banner__title'>RethinkDB</h1>
-      </div>
-      <div className='empty__message'>
-        <h2>No database connections added.</h2>
-        <p>Click the "+" to add a RethinkDB connection.</p>
-        <Link to='/newConnection'>
-          <Icon type='plus' size={32} />
-        </Link>
-      </div>
-    </StyledHome>
+    <Fragment>
+      <AppHeader />
```

Comment:
@gabor-boros what is this for?

Reply:
This change is needed because we need to have a header which can be moved by the user. If we do not add this header the only thing what we can do is to set the dragging for the whole window and its full content which means ex: that the user can move the window by dragging the logo which is unfortunate.